test/unit/capacityEngine.js

```diff
@@ -286,219 +282,6 @@ describe('Capacity Engine tests', function () {
     });
   });
 
-  describe('calculateAvailableCapacity', function () {
-    it('should calculate available capacity correctly when all tranches are usable', function () {
-      const trancheCapacities = [BigNumber.from(100), BigNumber.from(200), BigNumber.from(300)];
-      const allocations = [BigNumber.from(50), BigNumber.from(100), BigNumber.from(150)];
-      const firstUsableTrancheIndex = 0;
-
-      const result = calculateAvailableCapacity(trancheCapacities, allocations, firstUsableTrancheIndex);
-
-      expect(result.toString()).to.equal('300'); // (100-50) + (200-100) + (300-150) = 300
-    });
```

Comment:
since these functions are now in helpers.js, I've also moved the relevant unit tests to helpers.js unit tests